src/Event/DatasetIndexSubscriber.php

```diff
@@ -119,6 +120,12 @@ public function populateCalculatedFields(TransformEvent $event)
                 $document->set('collectionEndDate', $collectionEndDate->format('Y-m-d H:i:s'));
             }
         }
+
+        // For DIF only, set DOI to a DOI with empty string as the doi value.
+        if (false === $dataset->hasDatasetSubmission()) {
+                $doi = new DOI('');
```

Comment:
You don't need to do this, you can get the DOI from the dataset ($dataset->getDoi())

Reply:
indeed correct, changed.